spec/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require "bundler/setup"
+require "topological_inventory/ansible_tower/collector"
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = ".rspec_status"
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect
+  end
+end
+
+spec_path = File.dirname(__FILE__)
+Dir[File.join(spec_path, "support/**/*.rb")].each { |f| require f }
+
+#
+# You can add local requires to /lib/mock/require.dev.rb
+#
+require_dev_path = File.join(spec_path, "../lib", "require.dev.rb")
+require require_dev_path if File.exist?(require_dev_path)
```

Comment:
@slemrmartin this is handled by our bundler plugin in the main Gemfile in a much more flexible way, please remove this

Reply:
@agrare According to readme, bundler-inject is able override gems, how to add local `require "something"` like "pry-byebug" by this gem?. I don't want to explicitly write "require 'pry-byebug'" every time I need to debug something

Reply:
@slemrmartin bundler-inject really breaks down into two things
1. It pulls in extra gemfiles (e.g. from .bundler.d/*.rb and ~/.bundler.d/*`
2. It adds the extra DSL methods `override_gem` and `ensure_gem` which can be used in those extra gemfiles

But that doesn't mean that you have to only use `override_gem`.  You can just `gem 'pry-byebug'` in your bundler.d/ gemfile.

Personally I have `ensure_gem 'pry-byebug'` in my global `~/.bundler.d/pry.rb` file.  This works like `gem 'pry-byebug'` if pry-byebug isn't already in the applications Gemfile, or if it is it works like `override_gem 'pry-byebug'` and will override the other gem options.

Reply:
@agrare yes I added `ensure_gem 'pry-byebug' `, but it's only code for bundler, doesn't add `require` for runtime as this code does (at least on my pc debugging doesn't work without `require`).

Reply:
That sounds like an issue we should fix in bundler-inject rather than add a completely new thing, did you try adding a `:require => 'pry-byebug'` to the ensure_gem?

cc @Fryguy

Reply:
@agrare yes, but it doesn't work, I think it defines dependencies between gems, it doesn't load classes at runtime.

Reply:
@agrare it's not needed by this PR, removing. This feature is missing in all repos

Reply:
@slemrmartin can you open an issue on https://github.com/ManageIQ/bundler-inject then, it should work.